### DISCUSSION TITLE
How can I migrate browser from one realm to another

### DISCUSSION THREAD

**Discussion by Bhavik237** [keycloak/keycloak#21447] [Q&A] [answered]
Hello ,everyone 
I have created custom browser flow in one realm ,I want same flow in my new realm 
How should I approach it ??
I have tried importing/exporting json it didn't work...

Thanks

**Comment by sohaibaumer** [comment, marked as answer]
To replicate a custom browser flow from one realm to another in Keycloak, you can:
Manually recreate the flow in the new realm by configuring the authentication providers, actions, and flow sequence.
If the JSON export/import method doesn't work, try using a script or Keycloak extensions to automate the flow setup.

**Reply by christian-2** [reply]
(If the JSON export/import import method doesn't work) I'd perhaps use export and then `jq` to transform the JSON into a script that repeatedly called `kcadm.sh -r $other_realm ...`

**Reply by Bhavik237** [reply]
Thanks I have Done it through key-cloak API call , took 31 API calls though. );
